gymnasium/dev_wrappers/clip_action.py

```diff
@@ -0,0 +1,41 @@
+"""Wrapper for clipping actions within a valid bound."""
+import jumpy as jp
+
+import gymnasium as gym
+from gymnasium import spaces
+from gymnasium.core import ActType
+
+
+class ClipActionV0(gym.ActionWrapper):
```

Comment:
@RedTachyon For the vectorising method, do you want wrappers to inherit from `Lambda` wrapper version?